tutorials/monitoring/setup/index.md

```diff
@@ -0,0 +1,208 @@
+
+NOTE: You can use an existing ServiceControl instance if it is version 1.43 or above.
+
+In the ServiceControl Management Utility, click **+ New** and select **ServiceControl instance** from the list of options.
+
+![ServiceControl Management Utility - Add new ServiceControl instance](scmu-add-new-sc-instance.png)
+
+Scroll down to the **Transport Configuration** section. Select the transport that your NServiceBus system runs on. Depending on the transport you may be required to add an additional connection string.
+
+Under **Audit Forwarding** select either On or Off. If you are not currently using an audit queue for anything, you can safely turn this Off. See [audit fowarding](/servicecontrol/errorlog-auditlog-behavior.md) for more information.
+
+Click the Add button. Your ServiceControl instance will start and be listed on the main page of the ServiceControl Management Utility.
+
+![ServiceControl Management Utility - List instance - ServiceControl instance](scmu-list-sc-instance.png)
+
+The listing for the ServiceControl instance includes a URL link. This URL will be needed when installing ServicePulse below.
+
+NOTE: Creating the ServiceControl instance will also create the audit and error queues if they did not already exist. By default these are called _audit_ and _error_ respectively.
+
+
+### Create Monitoring Instance
+
+A Monitoring instance collects data from the monitoring queue and aggregates information from all of the endpoints in the system.
+
+In the ServiceControl Management Utility, click **+ New** and select **Monitoring instance** from the list of options.
+
+![ServiceControl Management Utility - Add new Monitoring instance](scmu-add-new-mon-instance.png)
+
+Scroll down to the **Transport Configuration** section. Select the transport that your NServiceBus system runs on. Depending on the transport selected you may be required to add an additional connection string.
+
+Click the Add button. Your Monitoring instance will start and be listed on the main page of the ServiceControl Management Utility.
+
+![ServiceControl Management Utility - List instance - Monitoring instance](scmu-list-mon-instance.png)
+
+The listing for the Monitoring instance includes a URL link. This URL will be needed when installing ServicePulse below.
+
+NOTE: Creating the Monitoring instance will also create the monitoring queue if it did not already exist. By default this queue is called _Particular.Monitoring_. 
+
+
+
+### Install ServicePulse
+
+ServicePulse is a web application for production monitoring and recoverability. It connects to a Monitoring instance to display monitoring data and to a ServiceControl instance to display recoverability data.
+
+Download and run the latest ServicePulse installer. 
+
+NOTE: If you already have ServicePulse running on your machine you will need to uninstall it and download the latest version in order to be able to configure ServicePulse to connect to a ServiceControl Monitoring instance.
```

Comment:
> download the latest version

I think we should say explicitly what is the minimum version supporting this, rather than use generic things like "latest version". I propose to replace with

If you already have ServicePulse running on your machine ensure it's version X or higher. Otherwise, uninstall it and download the latest version in order to be able to configure ServicePulse to connect to a ServiceControl Monitoring instance.

Reply:
The problem here is that even if you have 1.10 installed, if you didn't configure a monitoring instance, you need to uninstall it and re-install it.

Reply:
Sp the instruction is always to uninstall and reinstall even if I know I have the latest version. Not clear.